go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/kubernetes-sigs/kube-storage-version-migrator
```

Comment:
switch to "sig.k8s.io"?

Reply:
I will do that in a separate PR.

Reply:
@sanchezl it's a small thing and it's been two months.

Reply:
> @sanchezl it's a small thing and it's been two months.

there's a PR open https://github.com/kubernetes-sigs/kube-storage-version-migrator/pull/46 .